src/ui/Link/Link.scss

```diff
@@ -0,0 +1,30 @@
+@import '../../styles/_config';
+
+.link--token {
```

Comment:
What just the link does? If nothing, we can name this as `link`. The modifier should be used to modify some component/style that we already have.